s3tests_boto3/functional/test_s3.py

```diff
@@ -2817,6 +2817,53 @@ def test_post_object_upload_size_below_minimum():
 
     r = requests.post(url, files=payload, verify=get_config_ssl_verify())
     assert r.status_code == 400
+    assert r.error_code == 'EntityTooSmall'
+
+def test_post_object_upload_size_rgw_chunk_size_bug():
+    # TODO: if this value is different in Teuthology runs, this would need tuning
```

Comment:
the rgw suite does vary the *stripe* size, but the chunk size stays constant at 4M: https://github.com/ceph/ceph/blob/main/qa/suites/rgw/verify/striping%24/stripe-greater-than-chunk.yaml

Reply:
Thanks, linked